src/coreclr/src/md/compiler/mdperf.cpp

```diff
@@ -1,95 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-//*****************************************************************************
-// MDperf.cpp
-//
-
-//
-// This file provides Compiler Support functionality in metadata.
-//*****************************************************************************
-
-#include "stdafx.h"
-
-#include "mdperf.h"
-
-#ifdef FEATURE_METADATA_PERF_STATS
```

Comment:
If you delete this, `FEATURE_METADATA_PERF_STATS` will become broken (or even more broken than it is today). It should be deleted from everywhere.

Reply:
It may be best to do it in separate PR.